superset/security/manager.py

```diff
@@ -726,7 +726,8 @@ def create_custom_permissions(self) -> None:
         self.add_permission_view_menu("can_csv", "Superset")
         self.add_permission_view_menu("can_share_dashboard", "Superset")
         self.add_permission_view_menu("can_share_chart", "Superset")
-        self.add_permission_view_menu("can_view_and_drill", "Dashboard")
+        self.add_permission_view_menu("can_view_query", "Dashboard")
+        self.add_permission_view_menu("can_view_table", "Dashboard")
```

Comment:
Name could be a bit confusing potentially as "table" refers to different things in superset (the table viz, the database physical tables).  I guess with`Dashboard.can_view_table` at least we know that we're in the context of a `Dashboard.`l. It could be `Dashboard.can_view_chart_as_table`